citydb-tiling/src/main/java/org/citydb/tiling/options/DimensionScheme.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * 3D City Database - The Open Source CityGML Database
+ * https://www.3dcitydb.org/
+ *
+ * Copyright 2013 - 2024
+ * Chair of Geoinformatics
+ * Technical University of Munich, Germany
+ * https://www.lrg.tum.de/gis/
+ *
+ * The 3D City Database is jointly developed with the following
+ * cooperation partners:
+ *
+ * Virtual City Systems, Berlin <https://vc.systems/>
+ * M.O.S.S. Computer Grafik Systeme GmbH, Taufkirchen <http://www.moss.de/>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.citydb.tiling.options;
+
+import com.alibaba.fastjson2.annotation.JSONField;
+import com.alibaba.fastjson2.annotation.JSONType;
+import org.citydb.database.adapter.DatabaseAdapter;
+import org.citydb.database.geometry.GeometryException;
+import org.citydb.database.srs.SpatialReference;
+import org.citydb.database.srs.SrsException;
+import org.citydb.database.util.SrsHelper;
+import org.citydb.model.geometry.Coordinate;
+import org.citydb.model.geometry.Envelope;
+import org.citydb.model.geometry.Point;
+import org.citydb.tiling.TileMatrix;
+import org.citydb.tiling.TilingException;
+import org.citydb.tiling.TilingScheme;
+import org.citydb.tiling.encoding.PointReader;
+import org.citydb.tiling.encoding.PointWriter;
+
+import java.sql.SQLException;
+import java.util.Optional;
+
+@JSONType(typeName = "Dimension")
+public class DimensionScheme extends TilingScheme {
+    private Dimension width;
+    private Dimension height;
+    @JSONField(serializeUsing = PointWriter.class, deserializeUsing = PointReader.class)
+    private Point gridPoint;
```

Comment:
The gridPoint parameter not available in CLI?

Reply:
No

Reply:
Additional information: When using the `Dimension` tiling scheme, the tiles are automatically aligned with the database CRS. For example, when using 2x2km as width and height, the lower-left x-values of the tiles will be ..., -2000, 0, 2000, 4000, 6000, 8000, ... (and same for the y-values in this example). Only those tiles will be exported which cover the user-defined or auto-computed extent of the requested features, of course.

The `gridPoint` attribute allows a user to define a fixed point for tile alignment. For instance, if the user defines `(750,750)` as grid point in the above example, the the lower-left x-values would be ..., -1250, 750, 2750, 4750, ...

Reply:
OK. seems not very necessary to add it in CLI.